tests/python/driver/tvmc/test_transform.py

```diff
@@ -70,5 +70,55 @@ def test_layout_transform_convert_layout_pass_args(relay_conv2d, monkeypatch):
     )
 
 
+def test_layout_transform_to_mixed_precision_pass_args(relay_conv2d, monkeypatch):
+    """
+    Check the mixed precision arugments which are expected when
+    mixed precision arguments are provided.
+    """
+    mock_mixed_precision = MagicMock()
+    mock_mixed_precision.return_value = tvm.driver.tvmc.transform.MixedPrecision([])
```

Comment:
This seems to do a good job of checking the graph transformation function is called, do we have any tests which call into `tvmc.transform.MixedPrecision` to test it behaves as we expect?

Reply:
May be a bit tricky. We need to craft a graph and inspect the result ops. Let me try what we can do here.

Reply:
Managed with a testcase that pass through ToMixedPrecision and checks the dtypes across ops.